backend/src/test/java/gov/cdc/usds/simplereport/api/TestResultTest.java

```diff
@@ -21,12 +22,14 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
 
 @WithSimpleReportStandardUser // hackedy hack
 class TestResultTest extends BaseApiTest {
 
   @Autowired private TestDataFactory _dataFactory;
   @Autowired private OrganizationService _orgService;
+  @MockBean private SmsService _smsService;
```

Comment:
Is it possible to add a test to verify that SMS is working by creating a patient with testResultDelivery set? It may also be more appropriate to add it in TestOrderServiceTest, I'm not sure.

Reply:
Hmmm... yeah I think I could check that the MockBean has `send` called in `TestOrderServiceTest`, I will add that!